bats_test/create-drive/drive-name-validation.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# Assertions
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+@test "create drive with empty name" {
+    run -1 bash -c "yarn ardrive create-drive --dry-run -n '' -w $WALLET"
+
+    assert_line -n 0 "Error: Required parameter driveName wasn't provided!"
```

Comment:
You should also test for the exit status code (if possible)

Reply:
It's testing for exit status 1 on `run -1`